internal/flink/command.go

```diff
@@ -1,17 +1,21 @@
 package flink
 
 import (
+	"context"
 	"fmt"
 
 	"github.com/spf13/cobra"
 
+	cmfsdk "github.com/confluentinc/cmf-sdk-go/v1"
+
 	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
 	"github.com/confluentinc/cli/v4/pkg/config"
 	"github.com/confluentinc/cli/v4/pkg/featureflags"
 )
 
 type command struct {
 	*pcmd.AuthenticatedCLICommand
+	isMDSAuth bool
```

Comment:
I think it's cleaner to remove this and use `c.Config.IsOnPremLogin()` in `createContext()`